bin/copyRawDataToPrmTiming.sh

```diff
@@ -187,21 +187,30 @@ fi
 #
 # Make sure to use an account for cron jobs and *without* write access to prm storage.
 #
-if [[ "${ROLE_USER}" != "${ATEAMBOTUSER}" ]]
+if [[ "${ROLE_USER}" != "${DATA_MANAGER}" ]]
```

Comment:
The actual copying of data to prm is the only thing that requires write access to prm. Monitoring that process requires only read access and to prevent accidental data corruption on prm limiting the number of users or cron jobs that run with write access is best practice. Hence I suggest to run this as the ateambot user just like the ```notifications.sh``` script.

Reply:
The copyRawDataToPrm.sh is executed by the gd-dm, hence is thought it would be nice the use the same user for the CopyRawDataToPrmTiming.sh. i'll change it back.
Thanks for looking at my pull request